pages/open-positions.md

```diff
@@ -9,6 +9,24 @@ We're briefly pausing accepting new applications while we evolve how 18F plans f
 
 If you'd like to learn about open roles as they come available, you can sign up for the biweekly [18F Newsletter](https://18f.gsa.gov/#newsletter).
 
+## Opportunities with 18F partners
+We’re currently supporting other agencies’ recruiting efforts to build digital service and innovation teams. Below you’ll find a list of the current positions along with their associated agencies. For all of these opportunities, please note:
+- Priority deadline consideration for these roles is Friday, September 16, 2016.
+- You must be in the D.C. area or willing to relocate to D.C. Unfortunately, we can’t provide relocation assistance.
+
+If you’re interested in these opportunities, please email join18f@gsa.gov and reference the position you are interested in. You may apply to more than one role.
+
+**Environmental Protection Agency (EPA) Roles**
+- [Cloud Architect/Infrastructure Engineer](https://pages.18f.gov/joining-18f/raas/epa/cloud-architect/) 
+- [Technincal Architect, TAPD](https://pages.18f.gov/joining-18f/raas/epa/tapd-technical-architect/) 
```

Comment:
@rooneywp When you have a sec: `Technincal` --> `Technical`

`18f-pages` sites use a different approach for linking; looking up an example elsewhere in this repo.